server/src/routes/twiml.ts

```diff
@@ -1,31 +1,31 @@
 import { Request, RequestHandler, Response } from 'express';
 import { twiml, validateExpressRequest } from 'twilio';
 import { ServerCredentials } from '../common/types';
-import { error } from '../utils/log';
+import { log } from '../utils/log';
 
 export function createTwimlRoute(
   serverConfig: ServerCredentials,
 ): RequestHandler {
   const { VoiceResponse } = twiml;
 
-  const logError = (msg: string) => {
-    error(`/twiml ${msg}`);
+  const logMsg = (msg: string) => {
+    log(`/twiml ${msg}`);
   };
 
   return function twimlRoute(req: Request, res: Response) {
     const requestIsValid = validateExpressRequest(req, serverConfig.AUTH_TOKEN);
 
     if (!requestIsValid) {
       const msg = 'Unauthorized Twilio signature';
-      logError(msg);
+      logMsg(msg);
```

Comment:
Why log instead of error these messages? Aren't these all failed requests?

Reply:
This is a good question, I would say errors should be emitted if they are the fault of the server code, but in this case it's the fault of the request. What do you think? cc @charliesantos

Reply:
We need to add loglevels and define what each level means e.g. info, warn, debug, error. For now, let us use `log` for consistency.